lib/streamlit/elements/dataframe_selector.py

```diff
@@ -47,7 +47,7 @@ def dataframe(
 
         Parameters
         ----------
-        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray, Iterable, dict, or None
+        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray, snowflake.snowpark.DataFrame, Iterable, dict, or None
```

Comment:
And `line_chart`, `bar_chart`, `area_chart`, `table` also need to get this docstring addition